scripts/env.sh

```diff
@@ -36,8 +36,10 @@ if [ "$SET_CI_VARS" ]; then
   export NODE_ENV=development
   export LOCAL_LOCALSTACK_PORT_RANGE="4666-4683"
   export LOCAL_HASURA_PORT=8087
+  export LOCAL_WEB_PORT=3007
   export LOCAL_POSTGRES_PORT=5437
   export LOCAL_WEB_PORT=3007
+  export LOCAL_SEED_ADDRESS=0x21DF875603Ba765a15F9ae20e633D2DAe42952d9
```

Comment:
is this still needed?

Reply:
@levity yes, this is needed. this is the fix for the bug in Circle CI's path filtering orb which interprets a string "0x21DF875603Ba765a15F9ae20e633D2DAe42952d9" as a number and breaks everything.